docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml

```diff
@@ -99,6 +113,287 @@
 
           <entry>Initialize new workunit repository</entry>
         </row>
+
+        <row>
+          <entry>graph &lt;wu&gt;</entry>
+
+          <entry>Generate an alternative representation of the graph with
+          execution details</entry>
+        </row>
+
+        <row>
+          <entry>activity &lt;wu&gt;</entry>
+
+          <entry><para>What activities are executed between a range of times
+          (in time order)</para><para>&lt;wu&gt; ["&gt;scope|mintime"]
+          ["&lt;scope|maxtime"] [threshold=n%]</para></entry>
+        </row>
+
+        <row>
+          <entry>hotspot &lt;wu&gt; [&lt;activity&gt;]</entry>
+
+          <entry>Find the hotspots for workunit (or one particular
+          activity)</entry>
+        </row>
+
+        <row>
+          <entry>critical &lt;wu&gt; &lt;activity&gt;</entry>
+
+          <entry>What activities are executed in order to execute
+          activity</entry>
+        </row>
+
+        <row>
+          <entry>depend &lt;wu&gt; &lt;activity&gt; <emphasis
+          role="white">....</emphasis> &lt;activity&gt;</entry>
+
+          <entry>Find the common paths between two activities</entry>
+        </row>
+
+        <row>
+          <entry>depend &lt;wu&gt; ?&lt;activity&gt;:startTime</entry>
+
+          <entry>Which dependencies take a large % of the start time for this
+          activity</entry>
+        </row>
+
+        <row>
+          <entry>help &lt;command&gt;</entry>
+
+          <entry>More help on a command</entry>
+        </row>
+      </tbody>
+    </tgroup>
+  </informaltable>
+
+  <para>If the<emphasis role="bold"> [CASSANDRASERVER=ip]</emphasis> option is
```

Comment:
Since we no longer support Cassandra as a System Data Store, this whole section should be removed.